learn/appchains/index.md

```diff
@@ -1,9 +1,20 @@
 ---
 title: Appchain
 description: Discover the features in EVM and Substrate appchains deployed through Tanssi, leveraging user-friendly and easy-to-deploy templates for enhanced capabilities.
-template: subsection-index-page.html
-hide: 
- - toc
- - feedback
+icon: simple-flathub
```

Comment:
This icon is for a brand (see [flathub](https://flathub.org/) 👀 ), so we shouldn't use it here